plugins/linear-genome-view/src/LinearGenomeView/components/util.ts

```diff
@@ -65,7 +65,8 @@ export function getCytobands(assembly: Assembly | undefined, refName: string) {
   return (
     assembly?.cytobands
       ?.map(f => ({
-        refName: assembly.getCanonicalRefName(f.get('refName')),
+        refName:
+          assembly.getCanonicalRefName(f.get('refName')) || f.get('refName'),
```

Comment:
Is this a typo? why f.get twice?

Reply:
getCanonicalRefName can return undefined, and i updated the typescript to reflect this as it did not before. this explicitly makes usages of getCanonicalRefName uses a fallback of f.get('refName') if for whatever reason the refname is not part of the assembly

Reply:
the alternative to this behavior is making getCanonicalRefName internally add the fallback, but it may be of interest to some cases to know when it returns undefined